app/models/user-store/user-store.ts

```diff
@@ -159,6 +159,10 @@ export const UserStoreModel = types
         }
       }
     }),
+    postLikerLandLoginAction: flow(function * () {
```

Comment:
I would like to keep the action name start with a verb, how about 
`handleAfterLikerLandSignIn`?